airbyte-integrations/connectors/source-facebook-marketing/setup.py

```diff
@@ -10,6 +10,7 @@
     "cached_property==1.5.2",
     "facebook_business==13.0.0",
     "pendulum>=2,<3",
+    "python-dateutil==2.8.2",
```

Comment:
Seems like we already have `pendulum`, why not use that?

Reply:
updated